docs/book/src/component-config-tutorial/testdata/project/api/v2/projectconfig_types.go

```diff
@@ -52,8 +52,7 @@ type ProjectConfig struct {
 
 	// ControllerManagerConfigurationSpec returns the configurations for controllers
 	cfg.ControllerManagerConfigurationSpec `json:",inline"`
-
-	ClusterName string `json:"clusterName,omitempty"`
```

Comment:
We need to came back this line?

Also, after came back we need to run `make all` again.